README.md

```diff
@@ -113,7 +112,7 @@ Currently the CLI detects CI environments where data is being sent from.
 We're collecting this information to improve the support on how we collect the metrics.
 
 If your CI environment is not in this list,
-contact <mailto:pulsesupport@codacy.com> to let us know about it.
```

Comment:
The link was being rendered like this:

![image](https://user-images.githubusercontent.com/60105800/102624401-df4a5a80-413b-11eb-8796-7bf51e30704f.png)